src/pages/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+import Header from '@site/src/components/Header';
+
+//import styles from "./modules/index.module.css";
+
+export default function Home() {
+  const { siteConfig } = useDocusaurusContext();
+  return (
+    <Layout
+      title={siteConfig.title}
+      description="Distribute Artifacts Across OCI Registries With Ease">
```

Comment:
I feel this is a copy-paste artifact? :P

Reply:
> I feel this is a copy-paste artifact? :P

Took a little help from my previous code. Will change it :D